docs/misc/changelog.rst

```diff
@@ -17,6 +17,7 @@ Breaking Changes:
 New Features:
 ^^^^^^^^^^^^^
 - Add `n_cpu_tf_sess` to model constructor to choose the number of threads used by Tensorflow
+- `VecNormalize` now supports being pickled and unpickled.
```

Comment:
You should also update the example in the documentation too (named "Mujoco: Normalizing input features")

Reply:
done